test/WebJobs.Script.Tests/Configuration/ExtensionsSystemOptionsSetupTests.cs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.WebJobs.Script.WebHost.Configuration;
+using Microsoft.Extensions.Configuration;
+using Microsoft.WebJobs.Script.Tests;
+using Moq;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Configuration;
+
+[Trait(TestTraits.Group, TestTraits.WebhookTests)]
+public class ExtensionsSystemOptionsSetupTests
+{
+    [Fact]
+    public void Configure_BindsNamedOptions()
+    {
+        var inMemorySettings = new Dictionary<string, string>
+        {
+            ["AzureFunctionsJobHost:extensions:mcp:system:webhookAuthorizationLevel"] = "Anonymous",
+            ["AzureFunctionsJobHost:extensions:eventGrid:system:webhookAuthorizationLevel"] = "System",
+            ["AzureFunctionsJobHost:extensions:myext:someOtherKey"] = "shouldBeIgnored"
+        };
+
+        var config = new ConfigurationBuilder()
+            .AddInMemoryCollection(inMemorySettings)
+            .Build();
+
+        var setup = new ExtensionSystemOptionsSetup(config);
+
+        var options = new ExtensionSystemOptions();
+        setup.Configure("mcp", options);
+        Assert.Equal("mcp", options.ExtensionName);
```

Comment:
I personally prefer using AwesomeAssertions (aka FluentAssertions)


```suggestion
        options.ExtensionName.Should().Be("mcp");
```

Reply:
Prefer to use the built ins.